.github/workflows/installer-pr.yml

```diff
@@ -95,3 +98,59 @@ jobs:
             ```
             ${{ steps.read.outputs.content }}
             ```
+  create-harvester-pr:
+    runs-on: ubuntu-latest
+    needs: generate-image-diff
+    steps:
+      - uses: jungwinter/split@v2
+        id: spliter
+        with:
+          msg: ${{ github.ref_name }}
+          separator: '-'
+      - name: Download container-diff result
+        uses: actions/download-artifact@v3
+        with:
+          name: diff-result
+      - name: Read diff result
+        id: read
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: ./diff-result.txt
+      - name: Remove unused files
+        run: rm diff-result.txt
+      - name: Clone harvester repo
+        uses: actions/checkout@v3
+        with:
+          repository: harvester/harvester
+          ref: ${{ steps.spliter.outputs._0 }}
+      - name: Update nvidia driver toolkit tag
+        run: |
+          sed -i "/tag:/ {N; /harvester-nvidia-driver-toolkit/ s/tag:.*\n/tag: ${{ github.ref_name }}\n/}" package/upgrade/addons/nvidia-driver-toolkit.yaml 
+      - name: Create harvester PR
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: ${{ secrets.BOT_TOKEN }}
+          commit-message: Bump nvidia-driver-toolkit ${{ github.ref_name }}
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          signoff: false
+          branch: bump-os-${{ github.ref_name }}
+          delete-branch: true
+          title: 'Bump OS ${{ github.ref_name }}'
```

Comment:
suggest changing the branch name and title to "Bump nvidia-driver-toolkit addon"

Reply:
Nice catch... fixed